src/frontend/citizen-portal/src/app/components/dispute-stepper/dispute-stepper.component.ts

```diff
@@ -248,7 +265,15 @@ export class DisputeStepperComponent implements OnInit, AfterViewInit {
         allCountsValid = allCountsValid && (valid || countForm.value.__skip);
       }
     });
-    return allCountsValid && (this.mode === this.DisputeFormMode.UPDATE || !this.isAllCountsSkipped);
+    return allCountsValid && (this.mode === this.DisputeFormMode.UPDATE || !this.isAllCountsSkipped) && (this.mode === this.DisputeFormMode.UPDATE || this.isSignatureFormValid);
+  }
+
+  private get isSignatureFormValid(): boolean {
+    if (this.signatureBoxForm?.value.signatory_type === this.SignatoryType.D) {
+      return this.signatureBoxForm?.get('disputant_signatory_name')?.valid;
+    } else if (this.signatureBoxForm?.value.signatory_type === this.SignatoryType.A) {
+      return this.signatureBoxForm?.get('agent_signatory_name')?.valid;
+    }
```

Comment:
/** For the scenario where signatory_type === this.SignatoryType.U */
else { return false; }

Reply:
That should not be necessary since the SignatoryType is nullable and is set to null if not selected. I realize now that we actually don't need SignatoryType.U at all. I will probably remove it from the model in a separate PR.